### PR TITLE
[Bug] Navigate to issues 3125

### LINKED ISSUE #3125: [Bug] NavigateTo issues
## Description

- When selecting navigateTo the ui says that one can only navigate to a page name and does not mention external url
- The docs do not mention clearly that one can navigate to an external url
- The action selector does not add an undefined directly when skipping the query params argument

### PR DESCRIPTION
**Pull request by kaushik94**
## Description
When selecting navigateTo the ui says that one can only navigate to a page name and does not mention external url
The action selector does not add an undefined directly when skipping the query params argument. This PR fixes these issues.

Fixes #3125 

## Type of change
- New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>

    // Code coverage diff between base branch:release and head branch: bug/navigate-to-issue-3125 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 51.71 **(-0.01)** | 32.95 **(-0.02)** | 29.66 **(0)** | 52.2 **(0)**
 :red_circle: | app/client/src/components/editorComponents/ActionCreator.tsx | 34.25 **(-0.17)** | 23 **(-0.22)** | 21.11 **(0)** | 33.5 **(-0.18)**
 :red_circle: | app/client/src/components/editorComponents/GlobalSearch/index.tsx | 62.7 **(0)** | 14.84 **(-2.32)** | 42.5 **(0)** | 62.13 **(-0.22)**</details>

### CODE REVIEW

File: app/client/src/components/editorComponents/ActionCreator.tsx [hunk 1]
```diff
@@ -482,6 +482,9 @@ const fieldConfigs: FieldConfigs = {
       return textGetter(value, 1);
     },
     setter: (value: any, currentValue: string) => {
+      if (!value) {
+        value = " ";
```
Comment (hetunandu) [inline app/client/src/components/editorComponents/ActionCreator.tsx:486]: need to set undefined here

**Review by hetunandu — CHANGES_REQUESTED** [review]

**Comment by hetunandu** [comment]
This fix is not working

recording


<details><summary>Browser metadata</summary>





```
Path:      /applications/60067b0a65f65e25ea829159/pages/60067b0a65f65e25ea82915b/edit
Browser:   Chrome 91.0.4472.101 on Mac OS 10.15.7
Viewport:  1680 x 946 @2x
Language:  en-GB
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-5140--appsmith-editor-ce.netlify.app%2Fapplications%2F60067b0a65f65e25ea829159%2Fpages%2F60067b0a65f65e25ea82915b%2Fedit&os=OS+X&os_version=Catalina&browser=Chrome&browser_version=91.0&resolution=1680x946&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-5140--appsmith-editor-ce.netlify.app/applications/60067b0a65f65e25ea829159/pages/60067b0a65f65e25ea82915b/edit) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60caf16ec1cbe30007af2ea8&commentID=60cb2ebbccff5fdd0eeeef7b&resolution=resolved)

**Review by hetunandu — CHANGES_REQUESTED** [review]
Fix does not work

**Comment by ayushpahwa** [comment]
PR is too old, causing breaking changes when merging. New PR with the same changes with the latest release raised.

https://github.com/appsmithorg/appsmith/pull/5532

**Pull request closed without merging**